x/ecocredit/server/marketplace/add_allowed_denom.go

```diff
@@ -0,0 +1,29 @@
+package marketplace
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	api "github.com/regen-network/regen-ledger/api/regen/ecocredit/marketplace/v1"
+	"github.com/regen-network/regen-ledger/x/ecocredit/marketplace"
+)
+
+// AddAllowedDenom is a gov handler method that adds a denom to the list of approved denoms that may be used in the
+// marketplace.
+func (k Keeper) AddAllowedDenom(ctx sdk.Context, p *marketplace.AllowedDenomProposal) error {
```

Comment:
This might be best to keep as is but if you update to `AllowDenom` above then this should be updated as well.

Reply:
[changed](https://github.com/regen-network/regen-ledger/pull/1072/commits/d701ec57ad94a754739581df1168d427fb24c47d#diff-05d10aea58d74ec9800c304d727d0ce7aba45265bd49ccf0c6cbd4a163cc01a7R13)